Графика для всех страниц

Inga-e opened the pull request:
добавила изображения в index и catalog

---
:mortar_board: [Графика для всех страниц](https://up.htmlacademy.ru/htmlcss/35/user/2166645/tasks/7)

:boom: https://htmlacademy-htmlcss.github.io/2166645-device-35/3/

Inga-e committed:
Добавляет изображения

Создала папку images и в ней подпапки catalog, footer, header.
Оптимизировала изображения и добавила в index и catalog

Inga-e committed:
Изменяет alt в footer по сайту

Предыдущий коммит был с добавлением всех изображений.
Этот сделала, чтобы продублировать файлы index и catalog

saimon395 commented on index.html:
```diff
@@ -77,7 +77,7 @@ <h1 class="visually-hidden">Интернет-магазин "Девайс"</h1>
           <h2 class="visually-hidden">Популярные товары</h2>
 
-          <img class="slider-card" src="#" alt="Бестселлеры" >
+          <img class="slider-card" src="images/quadrocopter.jpg" width="560" height="560" alt="Квадрокоптер" >
           <button class="slider-button slider-prev" type="button">
```
нужно картинки и для остальных слайдов добавить, причём я бы их частью слайдов сделал

saimon395 replied:
про остальные картинки , кроме слайдера и футера тоже не забывай, в частности про список .groups-list

saimon395 replied:
и ещё момент, адреса лучше указывай относительные, то есть начинай их с ./

Inga-e replied:
могу я добавить одну и ту же картинку на все слайды? что значит, сделать частью слайдов?

Inga-e replied:
я подумала, что картинки в .groups-list не контентные, поэтому их не добавила. Я сумела только svg по ним без фона скачать? Фон можно потом добавить в css?

Inga-e replied:
зачем, на что это влияет? а адреса, которые я указала, они не подойдут без ./?

saimon395 replied:
> могу я добавить одну и ту же картинку на все слайды? что значит, сделать частью слайдов?

не стоит, добавляй сразу правильные картинки для разных слайдов, в стайлгайде они есть, заодно и текст поправь ,
а по поводу того что значит частью слайдов, помести картинки внутрь .slider-pagination-item

и ещё сейчас заметил, у тебя там всё содержимое li в button обёрнуто, это не правильно, кнопку и всё остальное отдельно делай , хоть и в одном li

saimon395 replied:
> я подумала, что картинки в .groups-list не контентные, поэтому их не добавила. Я сумела только svg по ним без фона скачать? Фон можно потом добавить в css?

ну в целом да, можно и потом добавить, но вообще svg не обязательно фоном добавляют

saimon395 replied:
> зачем, на что это влияет? а адреса, которые я указала, они не подойдут без ./?

на то как путь считается, просто то как у тебя сейчас написано, это абсолютный путь, а с точкой в начале это относительный, сейчас особых проблем нет, потому что файловая структура достаточно простая, но даже так, если использовать какой нибудь сервер который положит твои файлы не в корень, то могут быть проблемы с тем что он что то найти не сможет, так что лучше сразу привыкай указывать относительные пути

Inga-e committed:
Добавляет файл со стилем

создала файл со стилем и подключила стиль в index

Inga-e committed:
Корректирует индекс и каталог по фото

Внесла все по комментариям и добавила относительные пути по фото в каталоге. Скорректировала разметку по слайдеру в индексе.

Inga-e committed:
Удаляет лишние строки в слайдере
